### PR TITLE
(doc) z/Install section fixes

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <igor.kazmyr@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)

This PR aims to fix 3 issues in the z/OS installation set of docs, namely; 

1. https://docs.zowe.org/stable/user-guide/install-zos/ - to fix links in the diagram.
2. https://docs.zowe.org/stable/user-guide/installandconfig/#planning-zos-installation - add a note about the API ML-specific requirement to the Planning z/OS installation section (bullet with the started tasks).
3. https://docs.zowe.org/stable/user-guide/install-zowe-smpe/ - fix the DASD required storage space in the table [In addition, the information in the JCL has to be fixed]. The current figures are incorrect (as a result of the UX study conducted in Broadcom).

:heart:Thank you!

### CODE REVIEW

**Comment by IgorCATech** [comment]
I might need assistance with fixing those links in the diagram. There is some specificity to the syntax, I believe. For example, the syntax for the link that eventually lands here https://docs.zowe.org/stable/user-guide/install-zos/install-zowe-smpe-zosmf-workflow.html is: 

`<area href="install-zowe-smpe-zosmf-workflow.html" alt="Install the Zowe SMP/E build with z/OSMF workflow" title="Install the Zowe SMP/E build with z/OSMF workflow" shape="rect" coords="250, 498, 391, 555" />`
 
The issue is that, despite `href="install-zowe-smpe-zosmf-workflow.htm"`, the URL includes an extra directory (or level) in it, which is `/install-zos`. The correct URL should be https://docs.zowe.org/stable/user-guide/install-zowe-smpe-zosmf-workflow.html  
 
How can this issue be fixed? 
Because of this issue, all of the links in the diagram, except https://www.zowe.org/download.html, are broken.

File: docs/user-guide/installandconfig.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ The following information is required during the installation process. Software
 - (If not using Docker) Zowe uses a zFS directory to contain its northbound certificate keys as well as a truststore for its southbound keys. Northbound keys are one presented to clients of the Zowe desktop or Zowe API Gateway, and southbound keys are for servers that the Zowe API gateway connects to.  The certificate directory is not part of the Zowe runtime so that it can be shared between multiple Zowe runtimes and have its permissions secured independently. 
 
 - Zowe has the following started tasks:
+
+   **Note**: API ML installation requires only `ZWESVSTC`, so you can skip `ZWESISTC` and `ZWESASTC`.
```
Comment (Joe-Winchester) [inline docs/user-guide/installandconfig.md:32]: I thought that `ZWESISTC` is required for x509 client side certificate authentication which APIML uses, and after PI3 will be a feature of the Zowe CLI that can route requests through APIML (that in turn calls to ZWESISTC).

Maybe there should be a markdown table that shows each STC and then what it's used for,  so you could show that APIML only need ZWESVSTC for basic auth and JSON web token authentication, .but that ZWESISTC is needed for x509.

**Review by Joe-Winchester — CHANGES_REQUESTED** [review]

File: docs/user-guide/installandconfig.md (hunk 1 above)
Comment (IgorCATech) [reply docs/user-guide/installandconfig.md:32]: Thank you for reviewing the note! Unfortunately, I'm not an API ML expert but I can check with the team that is responsible for it. Maybe we can put together a table with the required started tasks.

**Comment by IgorCATech** [comment]
@nannanli Ashley, who would be able to help fix the broken links in the diagram?

File: docs/user-guide/installandconfig.md (hunk 1 above)
Comment (IgorCATech) [reply docs/user-guide/installandconfig.md:32]: Including @jalel01 , maybe he can point me in the right direction here. Who would be the right person to talk to about the issue?

**Comment by nannanli** [comment]
@IgorCATech Yes, let me work on them today. Thank you for catching them!

**Review by nannanli — APPROVED** [review]
I've fixed all the links in the roadmaps.

**Comment by IgorCATech** [comment]
Thank you, @nannanli!

File: docs/user-guide/installandconfig.md (hunk 1 above)
Comment (jalel01) [reply docs/user-guide/installandconfig.md]: Hi Igor, looks like this is resolved already - do you need any further assistance from the api ml squad?

Comment (IgorCATech) [reply docs/user-guide/installandconfig.md]: Thanks for getting back @jalel01 I just wanted to make sure the information is the notes is technically correct

Comment (balhar-jakub) [reply docs/user-guide/installandconfig.md]: The current state is technically correct.

Comment (Joe-Winchester) [reply docs/user-guide/installandconfig.md]: That's great if it's OK as-is.  I was mistaken, but I thought that x.509 client certificate support from APIML required ZSS which requires cross-memory.

**Comment by IgorCATech** [comment]
This PR is replaced by https://github.com/zowe/docs-site/pull/1803

**Pull request closed without merging**